functions/Invoke-DbaDiagnosticQuery.ps1

```diff
@@ -100,9 +149,12 @@ Then it will export the results to Export-DbaDiagnosticQuery.
         [Switch]$NoColumnParsing,
         [switch][Alias('Silent')]
         $EnableException
+        , [string]$OutputPath
+        , [switch]$ExportQueries
     )
 
     begin {
+        [int]$ProgressId = Get-Random
```

Comment:
cohercing is really not needed here

Reply:
I normally try to type any definition on first declaration to ensure it's correct and not implicit. Is that a bad practice?